packages/angular_devkit/build_angular/src/builders/jest/index.ts

```diff
@@ -8,6 +8,7 @@
 
 import { BuilderContext, BuilderOutput, createBuilder } from '@angular-devkit/architect';
 import { execFile as execFileCb } from 'child_process';
+import { promises as fs } from 'fs';
```

Comment:
```suggestion
import fs from 'node:fs/promises';
```

Reply:
Done. Out of curiosity, is there a meaningful difference between `{ promises as fs }` and `fs/promises` or is this just a style / consistency thing? Just wondering if there's some reasoning I'm not aware of.

Reply:
Effectively the same outcome for this case.  It provides more benefit when using named imports such as:
```
import { readFile } from 'node:fs/promises';
```